### DISCUSSION TITLE
Autorun

### DISCUSSION THREAD

**Discussion by piechade** [timmo001/system-bridge#1803] [Help] [answered]
Can I Autorun the system bridge without that the command line window opens first? 
![Snag_233abc](https://user-images.githubusercontent.com/1434005/176621319-f469d2b3-2b7e-48a8-b9b3-6f211f21be13.png)

**Comment by timmo001** [comment, marked as answer]
https://github.com/timmo001/system-bridge/pull/1633 will resolve this, but is waiting for a release upstream from sanic
